frontend/src/Pages/Student/Notice/NoticeDetails.jsx

```
@@ -0,0 +1,91 @@
+                            ) : null}
+                        </div>
+
+                        {/* Show embedded PDF if the file is a PDF */}
+                        {fileExtension === "pdf" ? (
+                            <iframe 
+                                src={fileUrl} 
+                                width="100%" 
+                                height="500px" 
+                                className="mt-2 border rounded"
+                                title="Notice Attachment"
+                            ></iframe>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)